src/conntuple/Manager.h

```diff
@@ -0,0 +1,33 @@
+// See the file "COPYING" in the main distribution directory for copyright.
```

Comment:
This header currently has no documentation or description at all.

Reply:
Added comments.